### DISCUSSION TITLE
idea-tips/debug/drop-frame-and-reset-frame

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#16] [Announcements]
# idea-tips/debug/drop-frame-and-reset-frame

大家在Debug程序的时候，是否遇到过因为“下一步”按太快，而导致跳过了想要深入分析的那段代码？是不是很想要有“回到上一步”这样的操作呢？在IDEA中就提供了一个帮助你回退代码的机会，但这个方法并不是万能的。好了，下面就来具体说说这个功能的使用！

https://www.didispace.com/idea-tips/debug/drop-frame-and-reset-frame.html

**Comment by huxiaoning** [comment]
很好很强大
